app/managers.py

```diff
@@ -0,0 +1,47 @@
+import os.path
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        path = os.path.join("..", "cinema.sqlite")
+        self._connection = sqlite3.connect(path)
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.cursor().execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list[Actor]:
+        actor_cursor = self._connection.cursor().execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        self._connection.commit()
```

Comment:
The `commit()` call after a SELECT query is unnecessary because SELECT does not modify the database. You can safely remove this line to avoid confusion.